### PR TITLE
add book publication information to LaTeX formats

### PR DESCRIPTION
**Pull request by nooraangelva**
Ref: https://github.com/cern-sis/issues-inspire/issues/58

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/literature/latex.py [hunk 1]
```diff
@@ -113,3 +122,29 @@ def get_dois(self, data):
         for doi_data in dois:
             doi_data["value"] = latex_encode(doi_data.get("value"))
         return dois or None
+
+    def get_book_publication_info(self, data):
+        if "book" in get_value(data, "document_type", []):
```
Comment (MJedr) [inline backend/inspirehep/records/marshmallow/literature/latex.py:127]: better check `if not` - then you can instantly return and remove one indentation level. Also, no need for default get value `[]`

File: backend/inspirehep/records/marshmallow/literature/latex.py [hunk 2]
```diff
@@ -113,3 +122,29 @@ def get_dois(self, data):
...
+            elif publisher:
+                return publisher
+        return missing
+
+    def get_isbn(self, data):
+        def hyphenate_if_possible(no_hyphens):
+            try:
+                return normalize_isbn(no_hyphens)
+            except ISBNError:
+                return no_hyphens
```
Comment (MJedr) [inline backend/inspirehep/records/marshmallow/literature/latex.py:144]: why inner function?

File: backend/inspirehep/records/marshmallow/literature/latex.py [hunk 3]
```diff
@@ -113,3 +122,29 @@ def get_dois(self, data):
...
+
+        if "book" in get_value(data, "document_type", []):
```
Comment (MJedr) [inline backend/inspirehep/records/marshmallow/literature/latex.py:146]: again, I would return here if it's not in `document_type`. Also, if I am correct only books have `isbns` - if that's the case you can get rid of this line and just check `if isbns`

File: backend/inspirehep/records/marshmallow/literature/latex.py (hunk 1 above)
Comment (nooraangelva) [reply backend/inspirehep/records/marshmallow/literature/latex.py]: At least when I test the latex_schema the ```[]``` default value is necessary.
```
if "book" not in get_value(data, "document_type"):
E       TypeError: argument of type 'NoneType' is not iterable
```

Comment (MJedr) [reply backend/inspirehep/records/marshmallow/literature/latex.py]: yes, you are right, sorry

**Review by MJedr — APPROVED** [review]

**Pull request merged**